Cargo.lock

```diff
@@ -10985,7 +10985,7 @@ version = "1.6.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1f559b464de2e2bdabcac6a210d12e9b5a5973c251e102c44c585c71d51bd78e"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
Don't know what this is about; I didn't do anything particular outside of using `diener`.

Reply:
a known bug in cargo:
* https://github.com/rust-lang/cargo/issues/5702
* https://github.com/rust-lang/cargo/issues/5529

Reply:
```suggestion
 "cfg-if 1.0.0",
```